grafana-plugin/src/models/loader/action-keys.ts

```diff
@@ -0,0 +1,5 @@
+export enum ActionKey {
+  IS_ADDING_NEW_COLUMN_TO_ALERT_GROUP = 'IS_ADDING_NEW_COLUMN_TO_ALERT_GROUP',
```

Comment:
I think we should simplify the format for example "ALERT_GROUP-ADD_NEW_COLUMN". The "IS_" prefix makes it difficult to reuse this enum for other stuff, for example an error of the same action.